doc/middleware-it-registratore-telematico/data-structures/data-structures.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: /poscreators/middleware-doc/italy/data-structures
+title: Data structures
+---
+
+## Data structures
+
+This chapter expands on the descriptions of the country-specific Data Structures, covered in the Chapter ["Data Structures"](../../general/data-structures/data-structures.md) of the General Part, with information applicable to the Italian market.
+
+### Receipt Request
+
+#### Single fields
+
+Fields from the receipt request that need special handling for the Italian market are listed below:
+
+| **Field name**               | **Data type**        | **Default Value Mandatory Field**                     | **Description**                                                                                                                                                                                                                                                                                                        | **Version** |
+|------------------------------|----------------------|-------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|
+| `cbTerminalID`               | `string (50)`        | Mandatory                                             | The unique identification of the input station/cash register/terminal within a ftCashBoxID                                                                                                                                                                                                                             | 1.3         |
+| `cbUser`                     | `string (50)`        | Mandatory                                             | Name (not ID) of the user who creates the receipt.                                                                                                                                                                                                                                                                     | 1.3         |
+| `cbReceiptReference`         | `string (50)`        | Mandatory | Used to connect all requests referring to the same business action.                                                                                                                                                                                                                                                    | 1.3         |
```

Comment:
The `cbReceiptReference` has to be unique in italy. `cbPreviousReceiptReference` is used for refunds.

IDK if theres a need to connect receipts like that in italy... If yes we could also use the `cbPreviousReceiptReference` for that.